docs/user-guide/quick-start.md

```diff
@@ -0,0 +1,53 @@
+# Kubernetes User Guide: Managing Applications: Quick start
+
+This guide will help you get oriented to Kubernetes and running your first containers on the cluster.
+
+## Launching a simple application
+
+Once your application is packaged into a container and pushed to an image registry, you’re ready to deploy it to Kubernetes.
+
+For example, [nginx](http://wiki.nginx.org/Main) is a popular HTTP server, with a [pre-built container on Docker hub](https://registry.hub.docker.com/_/nginx/). The [`kubectl run`](/docs/kubectl_run.md) command below will create two nginx replicas, listening on port 80.
+
+```bash
+$ kubectl run my-nginx --image=nginx --replicas=2 --port=80
+CONTROLLER   CONTAINER(S)   IMAGE(S)   SELECTOR       REPLICAS
+my-nginx     my-nginx       nginx      run=my-nginx   2
+```
+
+You can see that they are running by:
+```bash
+$ kubectl get po
+NAME             READY     STATUS    RESTARTS   AGE
+my-nginx-l8n3i   1/1       Running   0          29m
+my-nginx-q7jo3   1/1       Running   0          29m
+```
+
```

Comment:
Give the user some sense that the thing is running.  E.g.
`kubectl logs`.  So they get some sense of completion even if they can't make expose work.